### ISSUE TITLE
WooCommerce Code Reference

### ISSUE THREAD

**Issue by santanamic** [woocommerce/woocommerce#27330]
I noticed changes in the documentation. This is horrible compared to before. Woocommerce had the best documentation for an open source project. Now everything is an unproductive mess.

**Comment by peterfabian** [comment]
Hi, thanks for the feedback! Can you please be more specific about what you don't like about the new documentation? 
Unfortunately, the previous one we had hasn't been updated for quite some time, so we started using a new PHPDoc tool, but maybe we can make it useful to you again?

**Comment by starypop** [comment]
Hi @peterfabian!

I've got some improvement/fix ideas:

1. Restore a way to browse the source code of methods and clasess. In the previous version it was possible by clicking the name of the file that contains a method. Now I cannot find it, so the whole code reference is kinda useless for me.
2. Remove underline from links in the first column with method names, because there's no way to see the underscores.
3. Do not wrap method names in that column, or at least do not wrap it in the middle of a word.
4. Make the first column a lot wider. Now the last column with parameters description is to wide. And use smaller line-height than in the description column.

Here's a screenshot showing current columns:
![Zrzut ekranu 2020-08-18 o 10 16 51](https://user-images.githubusercontent.com/27000783/90488785-b0e05280-e13c-11ea-8c5a-8e6e644f687f.png)

**Comment by starypop** [comment]
It would be also nice to display WC version since a method has been deprecated.

Eg. https://woocommerce.github.io/code-reference/classes/WC-Abstract-Legacy-Order.html#method_get_product_from_item

**Comment by gregs-legs** [comment]
I entirely agree.  It is unreadable.  Moreover the links to the source code simply redirect you to more documentation.  You go around in circles but never actually get to the code.  A big step backwards.

**Comment by santanamic** [comment]
I agree with @starypop # 1. There is a lack of direct visualization for implementing class / methods/ functions/ templates.

Another thing is that woocommerce.js has some useful events however in the documentation they are not mentioned.

**Comment by peterfabian** [comment]
Bumping the priority as for an open source project, having a usable and good documentation is important.

**Comment by claudiosanches** [comment]
There's already a repository for it, we should move all issues related to the code reference to it's repository.
I'm going to move this issue there.

@starypop please feel free to open issues on https://github.com/woocommerce/code-reference 
Thanks for your suggestions.
